lib/kitchen/directions/bake_page_abstracts.rb

```diff
@@ -9,6 +9,21 @@ def self.v1(chapter:)
           abstract.prepend(child: "<h3 data-type='title'>#{I18n.t(:learning_objectives)}</h3>")
         end
       end
+
+      def self.v2(chapter:)
+        chapter.abstracts.each do |abstract|
+          ul = abstract.first('ul')
```

Comment:
Recommend `first!` b/c if the `ul` child is not found you want it to explode with a meaningful message, otherwise you'll get a weird error about `ul` being `nil` on the next line.